hoover/site/settings/common.py

```diff
@@ -152,3 +159,9 @@
 CELERY_BROKER_URL = os.getenv('SEARCH_AMQP_URL')
 
 SNOOP_FORWARD_HEADERS = ['Content-Disposition', 'Accept-Ranges', 'Content-Range', 'Content-Length']
+
+TUS_UPLOAD_DIR = os.path.join(base_dir, 'tus_upload')
+TUS_DESTINATION_DIR = os.path.join(base_dir, 'media', 'uploads')
```

Comment:
So this is the staging area where the uploaded files show up when they're done, or something else?

Same as above, let's move the storage outside `base_dir` by default.